modules/guides/pages/deleting-data.adoc

```diff
@@ -0,0 +1,244 @@
+= Deleting Data
+:description: How to delete documents in Couchbase.
+:page-pagination: full 
+:page-topic-type: guide
+:tabs:
+:github: Click the GitHub button icon:github[] to view this code in context.
+
+include::partial$example-attributes.adoc[]
+
+[abstract]
+{description}
+
+In situations where data is no longer needed, Couchbase Server provides a `remove` operation to delete a document from the database permanently. 
```

Comment:
Remove markup from `remove` thoughout this page where it's referring to the generic term for the operation, rather than a specific command or option